modules/KalturaSupport/components/playlistAPI.js

```diff
@@ -397,15 +397,15 @@
 				title = kWidget.sanitize( title );
 				description = kWidget.sanitize( description );
 
-				var thumbnailUrl = this.getThumbUrl(item) || item.thumbnailUrl || customData.thumbUrl;
+				var thumbnailUrl = item.thumbnailUrl || customData.thumbUrl || this.getThumbUrl(item);
```

Comment:
same for custom data, so you can make this generic.

Reply:
Can you spesify?

Reply:
`customData.thumbUrl`